addons/payment/data/payment_icon_data.xml

```diff
@@ -20,11 +20,6 @@
         <field name="image" type="base64" file="payment/static/img/diners_club_intl.png"/>
     </record>
 
-    <record id="payment_icon_cc_diners_club_intl" model="payment.icon">
```

Comment:
It's duplicated from https://github.com/odoo/odoo/blob/1d34e269dac2e0cafd85ad91a6240569b5b7282c/addons/payment/data/payment_icon_data.xml#L18-L21